client/reader/user-profile/style.scss

```diff
@@ -13,8 +13,19 @@
 
 		div.user-profile__wrapper {
 
+			.back-button {
+
+				@media ( max-width: 1300px ) {
+					padding: 24px 24px 0;
+				}
+
+				@media ( max-width: 600px ) {
+					padding: 24px 0 0;
+				}
+			}
+
```

Comment:
The `Stream` component is nested within the profile page and we can't easily apply styles from other parts of the Reader here.